lib/ansible/plugins/connection/ssh.py

```diff
@@ -5,8 +5,11 @@
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 
 from __future__ import (absolute_import, division, print_function)
+
```

Comment:
```suggestion
```

Unnecessary whitespace change; this boilerplate is not supposed to have a newline between those two lines.